Sources/ArcGISToolkit/Utility/SearchField.swift

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+
+/// A custom view implementing a SearchField. It contains a search button, text field, delete text button,
+/// and a button to allow users to hide/show the search results list.
+public struct SearchField: View {
```

Comment:
Just out of curiosity, why was this made public? It seems more like an implementation detail and not something that should have been exposed outside the toolkit.

Reply:
It was originally thought that the SearchField could be something that users could use separately from the rest of the Search component. I'm not sure it was ever tested at the time, though, nor if it's ever been used that way.

Reply:
Okay. Thanks!